egs/tedlium/s5_r2/local/run_segmentation_long_utts.sh

```diff
@@ -0,0 +1,264 @@
+bash -x steps/cleanup/segment_long_utterances.sh --cmd "$train_cmd" \
+  --stage $segment_stage \
+  --config conf/segment_long_utts.conf \
+  --max-segment-duration 30 --overlap-duration 5 \
+  --num-neighbors-to-search 1 --nj 80 \
+  exp/tri1b data/lang_nosp data/train_long data/train_reseg${affix} \
+  exp/segment_long_utts${affix}_train
+
+steps/compute_cmvn_stats.sh data/train_reseg${affix} \
+  exp/make_mfcc/train_reseg${affix} mfcc
+utils/fix_data_dir.sh data/train_reseg${affix}
+
+###############################################################################
+# Train new model on segmented data directory starting from the same model
+# used for segmentation. (tri2_reseg)
+###############################################################################
+
+# Align tri2b system with reseg${affix} data
+steps/align_si.sh  --nj 40 --cmd "$train_cmd" \
+  data/train_reseg${affix} \
+  data/lang_nosp exp/tri1b exp/tri1b_ali_reseg${affix}  || exit 1;
+
+# Train LDA+MLLT system on reseg${affix} data
+steps/train_lda_mllt.sh --cmd "$train_cmd" \
+  4000 50000 data/train_reseg${affix} data/lang_nosp \
+  exp/tri1b_ali_reseg${affix} exp/tri2_reseg${affix}
+
+(
+utils/mkgraph.sh data/lang_nosp exp/tri2_reseg${affix} \
+  exp/tri2_reseg${affix}/graph_nosp
+for dset in dev test; do
+  steps/decode.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+    exp/tri2_reseg${affix}/graph_nosp data/${dset} \
+    exp/tri2_reseg${affix}/decode_nosp_${dset}
+  steps/lmrescore_const_arpa.sh  --cmd "$decode_cmd" data/lang_nosp \
+    data/lang_nosp_rescore \
+     data/${dset} exp/tri2_reseg${affix}/decode_nosp_${dset} \
+     exp/tri2_reseg${affix}/decode_nosp_${dset}_rescore
+done
+) &
+
+###############################################################################
+# Train SAT model on segmented data directory
+###############################################################################
+
+# Train SAT system on reseg${affix} data
+steps/train_sat.sh --cmd "$train_cmd" 5000 100000 \
+  data/train_reseg${affix} data/lang_nosp \
+  exp/tri2_reseg${affix} exp/tri3_reseg${affix}
+
+(
+utils/mkgraph.sh data/lang_nosp exp/tri3_reseg${affix} \
+  exp/tri3_reseg${affix}/graph_nosp
+for dset in dev test; do
+  steps/decode_fmllr.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+    exp/tri3_reseg${affix}/graph_nosp data/${dset} \
+    exp/tri3_reseg${affix}/decode_nosp_${dset}
+  steps/lmrescore_const_arpa.sh  --cmd "$decode_cmd" data/lang_nosp \
+    data/lang_nosp_rescore \
+     data/${dset} exp/tri3_reseg${affix}/decode_nosp_${dset} \
+     exp/tri3_reseg${affix}/decode_nosp_${dset}_rescore
+done
+) &
+
+###############################################################################
+# Clean and segmented data
+###############################################################################
+
+segmentation_opts=(
+--max-junk-proportion=0.5
+--max-deleted-words-kept-when-merging=10
```

Comment:
can you explain why such a large value of this was necessary or helpful?
just for reference, the documentation about this say:
```
 help = "When merging segments that are found to be overlapping or "
                    "adjacent after all other processing, keep in the transcript the "
                    "reference words that were deleted between the segments [if any] "
                    "as long as there were no more than this many reference words. "
                    "Setting this to zero will mean that any reference words that "
                    "were deleted between the segments we're about to reattach will "
                    "not appear in the generated transcript (so we'll match the hyp).")
```